README.md

```diff
@@ -63,3 +63,86 @@ def perform_request(self, req)
 
 When using decorators, metrics are only tracked if the decorated function
 does not raise an error.
+
+### Context
+
+Statscor includes a context manager that can help measure latency and volume
+while using metric tags to classify their success & failure.  For example,
+suppose you are making a call to a remote service and wish to write a wrapper
+that collects latency, volume and failure metrics.
+
+With our knowledge about how the client library indicates errors we can make a context manager
+based on StatsContext:
+
+```
+from statsdecor.context import StatsContext
+
+class FoobarClientMetrics(StatsContext):
+    def __init__(self, tags=None):
+        tags = list(tags or [])
+        tags += ['caller:example_1']
+        super(ThingyStatsContext, self).__self__('thingy_client', tags=tags)
```

Comment:
`super(FoobarClientMetrics, self)`? or just `super()`?

Reply:
I think statsdecor is python2 compatible.  But I think `super()` by itself would work if it's python3.  (yay!!! Finally!!)